README.md

```diff
@@ -20,6 +20,17 @@ Then navigate to http://localhost:8080 in your browser
 
 ### Dependency management with pipenv
 
+We use pipenv locally, not in Docker. To verify pipenv is installed and working, run:
```

Comment:
The first part of this isn't technically true, refer to the `Dockerfile` to see how we use Pipenv to manage dependencies inside Docker. If one wants to modify the Pipfile however, they will need pipenv installed. So I think just removing the first statement will make this a-ok.

Reply:
Great point! I have removed it